config/routes.rb

```diff
@@ -10,4 +10,5 @@
   get 'ingredients/new/:recipe_id', to: 'ingredients#new', as: 'new_ingredient'
   post 'ingredients/new/:recipe_id', to: 'ingredients#create', as: 'create_ingredient'
   resources :public_recipes, only: [:index]
+  resources :general_shopping_list
```

Comment:
I think it could be a good idea to only add a route for "GeneralShoppingList#index", I think other actions are not going to be required